root/components/WorksList.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import WorkListEntry from '../static/scripts/common/components/WorkListEntry';
+
+import SortableTableHeader from './SortableTableHeader';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +checkboxes?: string,
+  +noRatings?: boolean,
+  +order?: string,
+  +seriesItemNumbers?: {+[number]: string},
+  +sortable?: boolean,
+  +works: $ReadOnlyArray<WorkT>,
+|};
+
+const WorksList = ({
+  $c,
+  checkboxes,
+  noRatings,
+  order,
+  seriesItemNumbers,
+  sortable,
+  works,
+}: Props) => (
+  <table className="tbl">
+    <thead>
+      <tr>
+        {$c.user_exists && checkboxes ? (
+          <th style={{width: '1em'}}>
```

Comment:
Unneeded style attribute.

Reply:
Why is it unneeded? It's in all other EntityList.js files, so I can remove it all as an extra commit, just not fully sure why that's the case :) Is it because checkboxes are always 1em and the table autoadjusts to them?

Reply:
> Why is it unneeded?

It was not in the corresponding TT file at least and I don’t see any need for it either.

> It's in all other EntityList.js files, so I can remove it all as an extra commit

Sure, we just missed that.

Reply:
It was in several TT files so I added to all lists for consistency, but it might be it's unneeded in all of them. I'll submit an extra commit here then.

Reply:
This column only contain a checkbox per row. IMHO, there is no need to set the column width. It would be find out if style has been set or unset at purpose for some templates, or if these were just inconsistent.